### PR TITLE
Fixed UI to use relative paths for ajax requests in order to work with `kube proxy`

### PR DESCRIPTION
**Pull request by kenfdev**
Signed-off-by: Ken Fukuyama <fukuyama@supersoftware.co.jp>

## Description

Because the ajax requests used the paths from the root, the gateway UI didn't work correctly via `kube proxy`. Using relative paths for the requests would solve this problem.
This is the [same fix](https://github.com/containous/traefik/pull/492) traefik has made, too.

## Motivation and Context

- [x] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))
People who use the gateway via `kube proxy` cannot use the UI currently.
This fixes issue #519.

## How Has This Been Tested?

I've spun up a kubernetes cluster with docker on mac. Deployed openfaas and checked

* Create
* Delete
* Invoke (downloaded an image, too)

both with and without `kube proxy`.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: gateway/assets/script/bootstrap.js [hunk 1]
```diff
@@ -143,7 +143,7 @@ app.controller("home", ['$scope', '$log', '$http', '$location', '$timeout', '$md
             var previous = $scope.functions;
 
             var cl = function(previousItems) {
-                $http.get("/system/functions").then(function(response) {
+                $http.get("../system/functions").then(function(response) {
```
Comment (alexellis) [inline gateway/assets/script/bootstrap.js]: is there any likelihood that nginx, caddy or some other ingress could block a relative path like "../"? cc @stefanprodan

Comment (stefanprodan) [reply gateway/assets/script/bootstrap.js]: It should be fine, if it works with kube proxy it should also work with nginx sub paths

Comment (alexellis) [reply gateway/assets/script/bootstrap.js]: Thanks for your advice. I'll merge.

**Comment by alexellis** [comment]
Will release 0.7.2 after the Angular upgrade in #531

**Pull request merged**